frontend/common/code-help/traits/traits-rest.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (envId, { FEATURE_NAME, FEATURE_FUNCTION, USER_ID, FEATURE_NAME_ALT }) => `# This will create a user in the dashboard if they don't already exist
+
+curl -X "POST" "https://edge.api.flagsmith.com/api/v1/identities/?identifier=winning2"
+     -H 'X-Environment-Key: ${envId}'
+     -H 'Content-Type: application/json; charset=utf-8'
+     -d $'{
+  "traits": [
+    {
+      "trait_value": 1000,
+      "trait_key": "gagan_1"
```

Comment:
Should we really be using Gagan's name as a trait key here? 🤔

Reply:
its just sample data - don't see this as an issue at all.